frontend/composables/use-utils.ts

```diff
@@ -8,7 +8,7 @@ export const useToggleDarkMode = () => {
 };
 
 export const useAsyncKey = function () {
-  return String(Date.now());
+  return `${Date.now()}-${Math.random().toString(36).slice(2)}`;
 };
```

Comment:
The other part of the fix. Requests fired quickly can have the same async key if we just rely on the date, causing some requests not to fire.